app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -2066,88 +2072,14 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
     }
 
     override fun selectMessages(messages: Set<MessageRecord>) {
-        handleLongPress(messages.first(), 0) //TODO: begin selection mode
-    }
-
-    // The option to "Delete just for me" or "Delete for everyone"
-    private fun showDeleteOrDeleteForEveryoneInCommunityUI(messages: Set<MessageRecord>) {
-        val bottomSheet = DeleteOptionsBottomSheet()
-        bottomSheet.recipient = viewModel.recipient!!
-        bottomSheet.onDeleteForMeTapped = {
-            messages.forEach(viewModel::deleteLocally)
-            bottomSheet.dismiss()
-            endActionMode()
-        }
-        bottomSheet.onDeleteForEveryoneTapped = {
-            messages.forEach(viewModel::deleteForEveryone)
-            bottomSheet.dismiss()
-            endActionMode()
-        }
-        bottomSheet.onCancelTapped = {
-            bottomSheet.dismiss()
-            endActionMode()
-        }
-        bottomSheet.show(supportFragmentManager, bottomSheet.tag)
-    }
-
-    private fun showDeleteLocallyUI(messages: Set<MessageRecord>) {
-        showSessionDialog {
-            title(resources.getQuantityString(R.plurals.deleteMessage, messages.count(), messages.count()))
-            text(resources.getString(R.string.deleteMessagesDescriptionDevice))
-            button(R.string.delete) { messages.forEach(viewModel::deleteLocally); endActionMode() }
-            cancelButton(::endActionMode)
-        }
+        selectMessage(messages.first(), 0) //TODO: begin selection mode
```

Comment:
TODO still req'd?

Reply:
Good question... This is a remnant from the old code, though I'm not sure what they meant by it.
I'll remove the comment